docs/release-date.md

```diff
@@ -16,3 +16,12 @@ To enforce that, transitive release date control is implemented. It works in the
   + If at least one released Study links to an entity (including transitive linking, e. g. Study linking to Analysis
     linking to File), this entity is considered to be released and becomes retrievable through the API.
   + Otherwise, the entity is not released and not retrievable through the API.
+  
+## Access to Entities before release date
+
+In certain cases the studies and its related entities should be accessible before release date for owners and 
+journals. And by default admins will be able to access the entities regardless of the release date.
```

Comment:
What (or who) exactly are journals? I haven't heard of this term before in this context